### DISCUSSION TITLE
Official Godot Android billing library now deprecated

### DISCUSSION THREAD

**Discussion by PickleJesus123** [godotengine/godot-proposals#8321] [Platforms]
As of November 1st 2023 (tomorrow), [Google Play billing v4 will be entirely deprecated](https://developer.android.com/google/play/billing/deprecation-faq). The problem is, the master branch of the [official Godot Google Play Billing plugin](https://github.com/godotengine/godot-google-play-billing) specifically targets this deprecated v4. However, there is another branch (billing-v5) which is up to date and working great, and it should become the new 'master' branch. It doesn't seem like the maintainers have checked in on the Android billing library in a while...

![billing](https://github.com/godotengine/godot-proposals/assets/12555805/11f89a37-aea2-4c1c-a19f-3ac2ab3a298a)

I would be happy to submit a PR that updates the documentation for the plugin, if someone would make billing-v5 the master branch

**Comment by Calinou** [comment]
Closing in favor of https://github.com/godotengine/godot-google-play-billing/issues/59, as this is a bug and not a feature proposal.
